x-pack/plugins/logs_shared/public/components/logging/log_entry_flyout/log_entry_actions_menu.test.tsx

```diff
@@ -24,6 +24,7 @@ const ProviderWrapper: React.FC = ({ children }) => {
 };
 
 describe('LogEntryActionsMenu component', () => {
+  const time = new Date().toISOString();
```

Comment:
**question:** I notice sometimes you used `moment` and some others the Date object to generate dates. Is there any reason for not using always the same constructor?

Reply:
Let's keep the change focused on the nano-seconds here, that's an implementation details and I don't want this PR to grow bigger 👌